src/google-analytics/GoogleAnalyticsService.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @implements {GoogleAnalyticsService}
+ * @memberof module:GoogleAnalytics
+ */
+class GoogleAnalyticsService {
+  constructor({ config }) {
+    this.analyticsId = config.GOOGLE_ANALYTICS_4_ID;
+
+    if (this.analyticsId) {
+      this.initializeService();
+    }
+  }
+
+  initializeService() {
+    // Create a queue, but don't obliterate an existing one!
```

Comment:
this commit seems very obvious, let's remove it, and also please review all next ones, thanks))

Reply:
@UvgenGen Please remove inline comments